### DISCUSSION TITLE
응용 프로그램 오류 관련 공지

### DISCUSSION THREAD

**Discussion by byungmeo** [byungmeo/GersangStation#49] [Announcements]
![image](https://github.com/user-attachments/assets/136bff9f-a410-4a9c-8235-b263fd208466)

금일 (24-12-26) 패치 이후 응용 프로그램 오류 관련 문의가 많이 오고 있습니다.
거상 스테이션 문제는 아니니 거상 공홈 참고하셔서 문제 해결하시기 바랍니다.

최근 개인 사정으로 바빠 오픈카톡 문의 답변이 느릴 수 있습니다.
거상 스테이션을 사용하실 때 오류가 발생하면 문의를 하시기 전에
자주 하는 질문, 거상 공홈, 커뮤니티 사이트 등을 참고하시는 것을 추천드립니다.

즐거되세요.

**Comment by qogus00** [comment]
감사합니다 ^^ 늘 감사히 생각하며 잘 사용하고 있습니다. 연말 마무리 잘 되시기 바래요 ~~
